### ISSUE TITLE
ranch [command] --help doesn't work

### ISSUE THREAD

**Issue by makebbekus** [goodeggs/platform#59]
When you execute `ranch` you're instructed that you can find help with `--help`, but it doesn't work :(

```
Use "ranch [command] --help" for more information about a command.
```

For example:
```
pick-api⏣ ranch version
6.6.0
pick-api⏣ ranch panic:rollback --help
Usage:
  ranch [command]

Available Commands:
  create         Create a new application
  debug          Debug
  deploy         Deploy the application
  deploy:conf    Deploy only configuration changes, not code changes
  env            Print the application environment
  env:set        Set environment variables
  env:unset      Un-set environment variables
  exec           Execute a command in an existing process
  info           Displays information about an application
  init           Initialize the .ranch.yaml file
  lint           Lint a ranch config
  logs           Tail the application logs
  panic:rollback Revert to a previous release
  panic:scale    Adjust the scale of a process
  ps             List an app's processes
  ps:stop        Stop a running process
  releases       List releases
  run            Run a one-off command
  run:detached   Run a detached one-off command
  update         Update the ranch CLI binary
  version        Prints the ranch CLI version

Flags:
  -a, --app="": application name (defaults to CWD)
  -f, --filename="": config filename (defaults to .ranch.yaml)
  -v, --verbose[=false]: verbose output

Use "ranch [command] --help" for more information about a command.
```

However, `ranch help [command]` works!

```
pick-api⏣ ranch help panic:rollback
Revert to a previous release

Usage:
  ranch panic:rollback <release> [flags]

Global Flags:
  -a, --app="": application name (defaults to CWD)
  -f, --filename="": config filename (defaults to .ranch.yaml)
  -v, --verbose[=false]: verbose output
```

**Comment by makebbekus** [comment]
I would've just changed the help text but it seems like it is auto-generated presumably from some other module?

**Comment by bobzoller** [comment]
released in v7.0.2
